flask_app.py

```diff
@@ -10,7 +10,9 @@
 # app setup
 smart_defaults = {
     'app_id': 'my_web_app',
-    'api_base': 'https://sb-fhir-stu3.smarthealthit.org/smartstu3/data',
+    # TODO: CapabilityStatement.rest.resource.search.param needs to provide 
+    # a required value type - config issue on our end for now
```

Comment:
What does this mean practically - is using flask still busted, but the url change gets us further? Or is this like a nice to have later?

Reply:
The old server this was pointed at no longer exists. The newer URL is the target we are looking at, but one of the searchparams it returns is missing a required field. Once that is fixed, this should work in the default config.

I've tested against HAPI's FHIR server, which works when that server is responding, but it's latency is pretty high, so I think fixing the one we control is preferrable.